src/homepageExperience/containers/GoWizard.tsx

```diff
@@ -154,7 +154,7 @@ export class GoWizard extends PureComponent<null, State> {
         return <WriteDataSql onSelectBucket={this.handleSelectBucket} />
       }
       case 5: {
-        return <ExecuteQuerySql bucket={this.state.selectedBucket} />
+        return <ExecuteQuerySql />
```

Comment:
Are we sure we don't need to populate `database` with the `bucket` here?

Reply:
Yes, the`database` is initialised in the "write-data" step: https://github.com/influxdata/ui/blob/a60db0a94c077ff49751490115a109cbc1df0400/src/homepageExperience/components/steps/go/WriteDataSql.tsx#L95